src/main/java/org/folio/rest/handlers/EventHandler.java

```diff
@@ -39,33 +39,38 @@ public EventHandler(PostgresClient postgresClient) {
   }
 
   public Future<String> handle(E event) {
+    log.debug("handle:: parameters event: {}", () -> logAsJson(event));
     return eventService.save(event)
       .compose(eventId -> updateUserSummary(event))
       .onComplete(result -> logResult(result, event));
   }
 
   public Future<String> handleSkippingUserSummaryUpdate(E event) {
+    log.debug("handleSkippingUserSummaryUpdate:: parameters event: {}",
+      () -> logAsJson(event));
     return eventService.save(event)
       .onComplete(result -> logResult(result, event));
   }
 
-  public Future<String> updateUserSummary(E event) {
+  private Future<String> updateUserSummary(E event) {
+    log.debug("updateUserSummary:: parameters event: {}", () -> logAsJson(event));
     return getUserSummary(event)
       .compose(userSummary -> userSummaryService.updateUserSummaryWithEvent(userSummary, event));
   }
```

Comment:
log result?

Reply:
We're only calling this method from `public Future<String> handle(E event)` and the next step after this call is logging: `.onComplete(result -> logResult(result, event));`. If we also add logging here, we'll be just doing it twice.